Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM python:3.6
+FROM python:3.9-buster
```

Comment:
Is this the python version running in prod? It may be useful to keep them in sync?

Reply:
Good point, let's keep this in sync. We're still running 3.6 in dev/prod.